kurbo v0.5.10 breaks compat with druid

I'm considering this an issue in kurbo, since according to semver a patch shouldn't break compatibility. There are nine places where this occurs, all with the same error: 

```
error[E0277]: the trait bound `kurbo::size::Size: std::convert::From<kurbo::vec2::Vec2>` is not satisfied
   --> /Users/code/.cargo/git/checkouts/druid-1a6b6c4d20db75fe/4728004/druid/src/widget/checkbox.rs:101:13
    |
101 |             RoundedRect::from_origin_size(Point::ORIGIN, Size::new(size, size).to_vec2(), 2.);
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::convert::From<kurbo::vec2::Vec2>` is not implemented for `kurbo::size::Size`
    |
   ::: /Users/code/.cargo/registry/src/github.com-1ecc6299db9ec823/kurbo-0.5.10/src/rounded_rect.rs:67:20
    |
67  |         size: impl Into<Size>,
    |                    ---------- required by this bound in `kurbo::rounded_rect::RoundedRect::from_origin_size`
    |
    = help: the following implementations were found:
              <kurbo::size::Size as std::convert::From<(f64, f64)>>
    = note: required because of the requirements on the impl of `std::convert::Into<kurbo::size::Size>` for `kurbo::vec2::Vec2`
```

Do note that this effects both version 0.4 on cargo and the latest from git.
```
error[E0277]: the trait bound `kurbo::size::Size: std::convert::From<kurbo::vec2::Vec2>` is not satisfied
   --> /Users/code/.cargo/registry/src/github.com-1ecc6299db9ec823/druid-0.4.0/src/widget/button.rs:119:13
    |
119 |             RoundedRect::from_origin_size(Point::ORIGIN, paint_ctx.size().to_vec2(), 4.);
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::convert::From<kurbo::vec2::Vec2>` is not implemented for `kurbo::size::Size`
    | 
   ::: /Users/code/.cargo/registry/src/github.com-1ecc6299db9ec823/kurbo-0.5.10/src/rounded_rect.rs:67:20
    |
67  |         size: impl Into<Size>,
    |                    ---------- required by this bound in `kurbo::rounded_rect::RoundedRect::from_origin_size`
    |
    = help: the following implementations were found:
              <kurbo::size::Size as std::convert::From<(f64, f64)>>
    = note: required because of the requirements on the impl of `std::convert::Into<kurbo::size::Size>` for `kurbo::vec2::Vec2`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "kurbo"
-version = "0.5.10"
+version = "0.5.11"
 authors = ["Raph Levien <raph.levien@gmail.com>"]
 license = "MIT/Apache-2.0"
 edition = "2018"

--- a/src/size.rs
+++ b/src/size.rs
@@ -295,6 +295,14 @@ impl From<Size> for (f64, f64) {
     }
 }
 
+//FIXME: remove for 0.6.0 https://github.com/linebender/kurbo/issues/95
+impl Into<Size> for Vec2 {
+    #[inline]
+    fn into(self) -> Size {
+        self.to_size()
+    }
+}
+
 #[cfg(test)]
 mod tests {
     use super::*;

